riski-extractor/config/config.py

```diff
@@ -51,6 +51,10 @@ class Config(BaseSettings):
         default=datetime.date.today().isoformat(),
         description="Start date for scraping (ISO format YYYY-MM-DD)",
     )
+    end_date: str = Field(
+        default=datetime.date.today().isoformat(),
```

Comment:
```suggestion
        default=None,
```
Make it optional? Then there would be an easy option to get all available future data that already exists (meetings etc.)